### PR TITLE
Small PR

### PR DESCRIPTION
**Pull request by sylenthuntress**
Not sure if I did good, it's my first PR, but I figured I'd give it a shot. Added pages for bientity conditions/actions, and documented a few specific ones.

### CODE REVIEW

File: docs/bientity_actions.md [hunk 1]
```diff
@@ -0,0 +1,11 @@
+---
+title: Actions
```
Comment (eggohito) [inline docs/bientity_actions.md]:
```suggestion
title: Bi-entity Actions
```

File: docs/bientity_actions.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
...
+date: 2021-10-06
+---
+# Bi-Entity Actions
+
+Bi-Entity Actions are a special type of action that acts upon two entities: The actor, and target. Which entity is the actor, or target, is largely dependent on the power type in question.
```
Comment (eggohito) [inline docs/bientity_actions.md]:
```suggestion
Bi-entity Actions operate on a `Pair<Entity, Entity>`; in simpler terms: an actor and a target. The actor and target is determined depending on the used power type, and can be swapped.
```

File: docs/bientity_conditions.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+---
+title: Conditions
```
Comment (eggohito) [inline docs/bientity_conditions.md]:
```suggestion
title: Bi-entity Conditions
```

File: docs/bientity_conditions.md [hunk 4]
```diff
@@ -0,0 +1,11 @@
...
+date: 2021-10-06
+---
+# Bi-Entity Conditions
+
+Bi-Entity Conditions operate in relation to two `LivingEntity`'s, an actor and a target. The actor and target are decided depending on the power type. Similar to entity conditions, these are available to be used in most powers in the `bientity_condition` field, which restricts when a power is active.
```
Comment (eggohito) [inline docs/bientity_conditions.md]:
```suggestion
Bi-entity Conditions operate on a `Pair<Entity, Entity>`, or in simpler terms: an actor and a target. The actor and target is determined depending on the used power type, and can be swapped. These are available to power types that provides a `bientity_condition` field, which restricts when a power is active.
```

File: docs/bientity_conditions/distance.md [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+title: Constant (Bi-Entity Condition)
+date: 2021-10-06
+---
+# Distance
+
+[Meta Condition](../meta_conditions.md).
+
+This condition is either always fulfilled or always not fulfilled. Mainly added as a backup in case a condition isn't optional in some place.
+
+Type ID: `origins:constant`
```
Comment (eggohito) [inline docs/bientity_conditions/distance.md]:
```suggestion
---
title: Distance (Bi-entity Condition)
date: 2021-10-07
---
# Distance

[Bi-entity Condition](../bientity_conditions.md)

Checks the distance between the target entity and the actor entity.

Type ID: `origins:distance`
```

File: docs/bientity_conditions/can_see.md [hunk 6]
```diff
@@ -0,0 +1,22 @@
+---
+title: Exists (Bi-Entity Condition)
```
Comment (eggohito) [inline docs/bientity_conditions/can_see.md]:
```suggestion
title: Can See (Bi-entity Condition)
```

File: docs/bientity_conditions/can_see.md [hunk 7]
```diff
@@ -0,0 +1,22 @@
...
+date: 2021-10-06
+---
+# Can See
+
+[Bi-Entity Condition](../bientity_conditions.md)
+
+Checks if you can see the entity targetted by the condition (e.g They are within your line of sight)
```
Comment (eggohito) [inline docs/bientity_conditions/can_see.md]:
```suggestion
Checks if the target entity is within the perspective angles (first person, front third person, back third person) of the actor entity.
```

File: docs/bientity_conditions/distance.md [hunk 8]
```diff
@@ -0,0 +1,29 @@
+---
+title: Constant (Bi-Entity Condition)
```
Comment (eggohito) [inline docs/bientity_conditions/distance.md]:
```suggestion
title: Distance (Bi-entity Condition)
```

**Review by eggohito — CHANGES_REQUESTED** [review]
There were a few inconsistencies in the documents. I've made suggestions that should fix the said inconsistencies. I've also reworded some bits in the documents to make it fit with the related documents (actions, conditions, etc.)

File: docs/bientity_conditions/distance.md (hunk 8 above)
Comment (sylenthuntress) [reply docs/bientity_conditions/distance.md]: Updated, my bad

**Review by eggohito — APPROVED** [review]
